python/feathub/processors/local/ast_evaluator/local_ast_evaluator.py

```diff
@@ -155,7 +155,10 @@ def eval_group_node(self, ast: GroupNode, variables: Optional[Dict]) -> Any:
 
     def eval_is_op(self, ast: IsOp, variables: Optional[Dict]) -> Any:
         left_value = self.eval(ast.left_child, variables)
-        is_none = left_value is None or np.isnan(left_value)
+        is_none = left_value is None
+        if isinstance(left_value, float) or isinstance(left_value, np.floating):
```

Comment:
Do we need to judge from `np.int8` to `np.float64`?

Reply:
Thanks for the suggestion. We can check if the `left_value` is an instance of `np.generic`. This takes care of all the numpy dtypes.